introduction/installation/sign-up-for-cloud.md

```diff
@@ -1,9 +1,34 @@
 ---
-title: Sign Up for Cloud
+title: Sign Up for GlareDB Cloud
 layout: default
 nav_order: 1
 parent: Installation
 grand_parent: Introduction / Getting Started
 ---
 
-## Sign Up for Cloud
+# Sign Up for GlareDB Cloud
```

Comment:
I'm not sure I understand the reason for/impact of the heading change

Reply:
The meta at the top of the file isn't a heading, so making the top-most heading of the page an `h2` was just a mistake